src/Symfony/Component/Console/Command/Command.php

```diff
@@ -284,6 +284,10 @@ public function setCode($code)
             throw new \InvalidArgumentException('Invalid callable provided to Command::setCode.');
         }
 
+        if (PHP_VERSION_ID >= 50400 && $code instanceof \Closure) {
```

Comment:
Is the PHP version check necessary? In PHP 5.3 the instanceof check will return false, as the `Closure` class doesn't exist.